Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -61,21 +61,18 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 PercExcl = NaN;
 MinimumtValue = NaN;
 
-if x.modules.asl.bMultiPLD || x.modules.asl.bMultiTE
+if x.modules.asl.bMultiPLD || x.modules.asl.bMultiTE || x.modules.asl.bDeltaM

```

Comment:
Not sure what `bDeltaM` is, but throughout ExploreASL it assumes that if `ASL4D` has only a single volume, this is a `deltaM` volume. Motion correction is disabled by default for a single volume.

BTW, why should ENABLE be disabled for DeltaM images? It should rather skip the subtraction.

Ah I think I now get it, you needed `bDeltaM` for GE eASL. But I would have created a separate issue for this then, because now parts of the pipeline use `bDeltaM` and other parts don't.

Reply:
bDeltaM == bContainsDeltaM . It tries to distinguish sequences containing deltaM from those containing control/labels.

Yes, the assumption that single-volume ASL4D has a deltaM is wrong - that's why I had to change this. GE multi-PLD has multiple deltaMs.

ENABLE needs to be disabled for deltaM as either it is a single volume, or it is multi-PLD and you can't start removing certain PLDs. DeltaM == true when the original sequence already contains the subtractions.

Yes - you got it right. We perhaps need an extra issue to use bDeltaM everywhere. But I needed to do this here - cause I couldn't split the multi-PLD eASL implementation to two issues - that wouldn't be possible to test. I can use bDeltaM everywhere - in this issue, or in a different issue?

Reply:
OK I agree.

I would have liked splitting this issue in multiple issues, e.g., `bDeltaM` should be a separate issue at all (not only halfly implementing it in this issue and fixing it in another ;)

But I'm fine either way. If you want to do this properly, search for `nVolumes>1` or something similarly, especially in the realign function.

Reply:
I have reviewed all use of `nVolumes` and either `bContainsDeltaM` is there or it doesn't need to be there. So we are OK to resolve this one.